src/components/Contacts/ContactDetails/ContactTasksTab/ContactTasksTab.tsx

```diff
@@ -252,7 +252,7 @@ export const ContactTasksTab: React.FC<ContactTasksTabProps> = ({
             })
           }
           style={{
-            height: `Max(calc(100vh - ${infiniteListRectTop}px), 260px)`,
+            height: `Max(calc(100vh - ${infiniteListRectTop}px), 500px)`,
```

Comment:
This change gives me double-scrolling. Also FYI you don't need `calc` inside of `max`. You can just do: `max(100vh - ${infiniteListRectTop}px, 500px)`.

Reply:
So this is the issue we've been having everywhere with Infinity Scroll. It has to have a set height to work. Which then creates double-scrolling. Do you have any ideas of a work around?

Reply:
I'm sorry, I misread what was changing. Yeah, I think it makes sense to have more than 1.5 tasks visible on small screens. Thanks for the fix!